app/src/main/java/com/example/triptracker/view/home/HomeScreen.kt

```diff
@@ -351,9 +351,11 @@ fun DisplayItineraries(
     homeViewModel: HomeViewModel,
     test: Boolean = false,
     tabSelected: HomeCategory = HomeCategory.TRENDING,
+    userProfileViewModel: UserProfileViewModel = viewModel()
 ) {
   var goodPadding = PaddingValues(0.dp, 0.dp, 0.dp, 70.dp)
-  val usermail = AmbientUserProfile.current.userProfile.value.mail
+  val userProfile = AmbientUserProfile.current.userProfile.value
+  val usermail = userProfile.mail
```

Comment:
Why did you split this line in two if you do not use the userProfile variable ? 
Could you please only keep a single line? 
Thank you

Reply:
Yes, of course ! 💮🌷